source/stylesheets/prism.css

```diff
@@ -371,3 +371,7 @@ div.code-toolbar > .toolbar span:focus {
 	content: attr(data-prompt);
 }
 
+table, th, td {
+   border: 1px solid #d5d2ff;
+   background: #dfe6f0;
+}
```

Comment:
This is the stylesheet for the prism code highlighter. Please use the available scss files to adjust other styling.

Reply:
Moved to layout.css